sos/report/plugins/cups.py

```diff
@@ -36,7 +36,8 @@ def setup(self):
         self.add_cmd_output([
             "lpstat -t",
             "lpstat -s",
-            "lpstat -d"
+            "lpstat -d",
+            "journalctl -u cups-browsed"
```

Comment:
Why not `self.add_journal(units="cups-browsed")`?

(or optionally enhance the `self.add_journal(units="cups")` to `self.add_journal(units=["cups", "cups-browsed"])`, but that will generate *one* file with `journalctl -u cups -u cups-browsed` output altogether. I dont know if it is better to split the logs or merge..).

Reply:
Actually I missed that, so I'm sending the patch again. Tested as well, and it works.